### DISCUSSION TITLE
Add safeguard to Core Maintainer Election

### DISCUSSION THREAD

**Discussion by trampster** [CommunityToolkit/Maui#1294] [General]
I'm concerned that the Core Maintainer Election might be subject to abuse and result in the selection of unsuitable candidates for the following reasons:

- The election is open to the anyone with a github account, not just community members, the election could be bombed by a group outside the community.
- If not enough candidates apply even though they are not suitable under the current Election text they will become Core Maintainers by default (top 3)
- Community voters may not vote according the guidelines in the text, resulting in people not meeting that criteria being selected.

In a discussion with Peter Morris on twitter he suggested that the existing Core Maintainers could veto unsuitable candidates if this was to occur. However as this is not mentioned in the election text if this was to occur it could result in significant community backlash.

I suggest that text be added to the election which states the following (or similar):

> The Core Maintainers reserve the right to veto unsuitable candidates.

**Comment by TheCodeTraveler** [comment]
Daniel. 

You've [insulted the .NET MAUI Community Toolkit team on Twitter](https://twitter.com/TheCodeTraveler/status/1678633720381685761?s=20) and you've also spread lies [that we have "not reached out privately to our top contributors"](https://twitter.com/TheCodeTraveler/status/1678657635846070272?s=20), [that "no prior experience is required"](https://twitter.com/TheCodeTraveler/status/1678864128801390594?s=20), and [that "no existing maintainers want the job"](https://twitter.com/TheCodeTraveler/status/1678863772147159041?s=20).

I hope you understand that my team cannot engage with you in good faith. To avoid further harassment to my team, I am blocking you from the Community Toolkit organization for 30 days. Upon further harassment, I will be forced to enact a permanent ban.

Your concerns above are unfounded. As you've highlighted above, quoting me, we are aware of the possibility and we reserve the right to veto unsuitable candidates.
